cmake/winml.cmake

```diff
@@ -189,6 +193,12 @@ set_target_properties(winml_lib_telemetry
   FOLDER
   ${target_folder})
 
+# Add deps
+add_dependencies(winml_lib_telemetry winml_sdk_cppwinrt)
+add_dependencies(winml_lib_telemetry winml_api)
+add_dependencies(winml_lib_telemetry winml_api_native)
+add_dependencies(winml_lib_telemetry winml_api_native_internal)
```

Comment:
Are these needed now? Were they not needed earlier?

Reply:
They are needed now so that the headers for the winrt component are available. Those headers are used to set the cppwinrt version.